src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.h

```diff
@@ -5,15 +5,66 @@
 
 struct EnBigpo;
 
-typedef void (*EnBigpoActionFunc)(struct EnBigpo*, GlobalContext*);
+typedef void (*EnBigPoActionFunc)(struct EnBigpo*, GlobalContext*);
+
+typedef struct EnBigpoFireParticle {
```

Comment:
We have been calling these effects rather than particles.

Reply:
Ah, alright. I thought Effect was reserved for action overlays called effects, will change